Kinvey-Xamarin-Android/Auth/AndroidNativeCredentialStore.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) 2016, Kinvey, Inc. All rights reserved.
+//
+// This software is licensed to you under the Kinvey terms of service located at
+// http://www.kinvey.com/terms-of-use. By downloading, accessing and/or using this
+// software, you hereby accept such terms of service  (and any agreement referenced
+// therein) and agree that you have read, understand and agree to be bound by such
+// terms of service and are of legal age to agree to such terms with Kinvey.
+//
+// This software contains valuable confidential and proprietary information of
+// KINVEY, INC and is subject to applicable licensing agreements.
+// Unauthorized reproduction, transmission or distribution of this file and its
+// contents is a violation of applicable laws.
+
+using System;
+using System.Collections.Generic;
+using Android.Accounts;
+using Android.Content;
+using Newtonsoft.Json;
+
+namespace KinveyXamarin
+{
+	/// <summary>
+	/// Android native credential store.
+	/// </summary>
+	public class AndroidNativeCredentialStore : NativeCredentialStore
+	{
+		private Android.App.Activity appActivity;
+		private Context appContext;
+		private AccountManager accountManager;
+
+		#region NativeStoreCredential implementation
+
+		/// <summary>
+		/// Initializes a new instance of the <see cref="T:KinveyXamarin.AndroidNativeCredentialStore"/> class.
+		/// </summary>
+		/// <param name="activity">App activity.</param>
+		public AndroidNativeCredentialStore(Android.App.Activity activity)
```

Comment:
@vinaygahlawat must we ask for the `activity`? Does it not suffice to ask for `context`?